opm/simulators/wells/MSWellHelpers.hpp

```diff
@@ -36,22 +37,27 @@ namespace Opm {
 
 namespace mswellhelpers
 {
-    // obtain y = D^-1 * x with a direct solver
+
+#if HAVE_UMFPACK
+    /// Applies umfpack and checks for singularity
     template <typename MatrixType, typename VectorType>
     VectorType
-    invDXDirect(const MatrixType& D, VectorType x)
```

Comment:
I could not comment on the line for some reason, basically the `invDxDirect` in the error message (line 61) should be changed following the change of the function name here?